tests/unit/TodoList.spec.ts

```diff
@@ -614,25 +614,25 @@ describe('TodoList - UI', () => {
     // init store
     store.state.tasks = [
       {
-        'task': {
```

Comment:
Same as App unit tests -- I did some cleanup here in tests that I originally broke, but there are actually very few changes in here.